purchase_operating_unit/models/stock_rule.py

```diff
@@ -0,0 +1,58 @@
+from odoo import _, models
+from odoo.exceptions import UserError
+
+
+class StockRule(models.Model):
+    _inherit = "stock.rule"
+
+    def _prepare_purchase_order(
+        self, product_id, product_qty, product_uom, origin, values, partner
```

Comment:
@BT-apadron This method signature is not OK. The new signature is:

`def _prepare_purchase_order(self, company_id, origins, values)`

Good news is that in our code, we are not using neither `product_id` nor `product_qty`, so we should be fine.

Reply:
Solved in https://github.com/brain-tec/operating-unit/pull/349